packages/core/test/components/Card/Content.test.tsx

```diff
@@ -16,22 +16,14 @@ describe('<Card />', () => {
     const imageUrl = 'LeftFoo.jpg';
     const wrapper = shallowWithStyles(<Content beforeImageSrc={imageUrl}>Sup</Content>);
 
-    expect(
-      shallow(wrapper.find(Row).prop('before') as any)
-        .find('img')
-        .prop('src'),
-    ).toBe(imageUrl);
+    expect((wrapper.find(Row).prop('before')! as any).props.src).toBe(imageUrl);
```

Comment:
does this work? so we dont need the `! as any`

`expect(shallow(wrapper.find(Row).prop('before')).prop('src')).toBe(imageUrl);`

looks like it had a shallow before

Reply:
@stefhatcher before we shallowed and asserted on the nested item, since it was wrapped in a div, this was possible.  But now Image is the top level item in the prop, and enzyme does not support asserting props on the wrapper itself.  So I assert on the passed in component.

Reply:
gotcha 👍